fixuid.go

```diff
@@ -281,20 +282,65 @@ func logInfo(v ...interface{}) {
 	}
 }
 
-func exitOrExec(runtimeUIDInt int, runtimeGIDInt int, argsWithoutProg []string) {
+func exitOrExec(runtimeUIDInt, runtimeGIDInt int, containerUser, oldGID, newGID string, argsWithoutProg []string) {
```

Comment:
think this actually needs to use the Real User ID to look stuff up as opposed to `containerUser` string